src/pages/index.js

```diff
@@ -166,7 +184,15 @@ const IndexPage = ({ data }) => {
                 </div>
               </div>
               <div className="video-wrapper">
-                <img src={tidbFeaturesGIF.publicURL} alt="TiDB-Feature" />
+                {browser.isMiBrowser || browser.isX5Browser ? (
+                  // if browser is MiBrowser or WeChat X5 Browser, show GIF for resolving z-index error
+                  <img src={tidbFeaturesGIF.publicURL} alt="" />
+                ) : (
+                  <video loop muted autoPlay poster={tidbFeaturesPNG.publicURL}>
```

Comment:
I think the poster is unnecessary here, since you have set the video autoplay.

Reply:
ok, i will remove the attribute.